test/util.js

```diff
@@ -513,3 +515,261 @@ function checkCursorPaginatedHistoryObject ({
     }
   })
 }
+
+/**
+ * @param {Object}   params
+ * @param {Object}   params.t - AVA test object
+ * @param {Object}   params.authorizationHeaders
+ * @param {String}   params.endpointUrl - tested endpoint
+ * @param {String}   [params.fetchEndpointUrl] - if not specified, fallback to endpointUrl
+ *   is the endpoint to hit to auto-generate test values
+ *   (useful to specify it for stats and history endpoints)
+ * @param {Function} [params.checkPaginationObject(t, obj)]
+ * @param {Object[]} params.filters
+ * @param {String}   params.filters[i].prop
+ * @param {Function} [params.filters[i].customGetValue(obj)]
+ *   custom getter to value with prop (by default obj[prop])
+ * @param {Function} [params.filters[i].customCheck(obj, value)]
+ *   if specified, will perform a custom check on that property
```

Comment:
It would be helpful to state which checks are performed by default.

Reply:
Indeed, I'll add explanations on that.